### PR TITLE
randoms without a seed

### PR DESCRIPTION
**Pull request by justin1121**
Fixes https://github.com/mortendahl/tf-encrypted/issues/274

### CODE REVIEW

File: operations/secure_random/test_secure_random.py [hunk 1]
```diff
@@ -9,15 +9,18 @@
 dirname = os.path.dirname(tfe.__file__)
 shared_object = dirname + '/operations/secure_random/secure_random_module.so'
 secure_random_module = tf.load_op_library(shared_object)
+seeded_random_uniform = secure_random_module.seeded_random_uniform
+random_uniform = secure_random_module.secure_random_uniform
```
Comment (mortendahl) [inline operations/secure_random/test_secure_random.py]: rename `secure_random_module.secure_random_uniform` to `secure_random_module.random_uniform` to match the seeded version?

File: operations/secure_random/test_secure_random.py [hunk 2]
```diff
@@ -9,15 +9,18 @@
...
+seed = secure_random_module.seed
 
 
-class TestSecureRandom(unittest.TestCase):
+class TestSeededSecureRandom(unittest.TestCase):
```
Comment (mortendahl) [inline operations/secure_random/test_secure_random.py]: `TestSeededRandomUniform`?

File: operations/secure_random/test_secure_random.py [hunk 3]
```diff
@@ -48,18 +51,65 @@ def test_invalid_max_min(self):
...
             minval = tf.constant(-2000, dtype=tf.int64)
             maxval = tf.constant(-1000, dtype=tf.int64)
 
-            output = secure_random_module.secure_random([2, 3], [1, 1, 1, 1, 1, 1, 1, 2], minval, maxval).eval()
+            output = seeded_random_uniform([2, 3], [1, 1, 1, 1, 1, 1, 1, 2], minval, maxval).eval()
 
             np.testing.assert_array_equal(output, expected)
 
 
+class TestSecureRandom(unittest.TestCase):
```
Comment (mortendahl) [inline operations/secure_random/test_secure_random.py]: `TestRandomUniform`?

File: tests/test_secure_random.py [hunk 4]
```diff
@@ -2,20 +2,20 @@
 
 import tensorflow as tf
 import numpy as np
-from tf_encrypted.operations.secure_random import secure_random
+from tf_encrypted.operations.secure_random import seeded_random_uniform, random_uniform, seed as seed_gen
```
Comment (mortendahl) [inline tests/test_secure_random.py]: hmm, why two test files?

File: tf_encrypted/operations/secure_random/secure_random.py [hunk 5]
```diff
@@ -40,4 +40,39 @@ def secure_random(shape, minval=0, maxval=None, dtype=tf.int32, seed=None, name=
     minval = ops.convert_to_tensor(minval, dtype=dtype, name="min")
     maxval = ops.convert_to_tensor(maxval, dtype=dtype, name="max")
 
-    return secure_random_module.secure_random(shape, seed, minval, maxval, name=name)
+    return secure_random_module.seeded_random_uniform(shape, seed, minval, maxval, name=name)
+
+
+def random_uniform(shape, minval=0, maxval=None, dtype=tf.int32, name=None):
+    """
+    Returns random numbers securely.
```
Comment (mortendahl) [inline tf_encrypted/operations/secure_random/secure_random.py]: `Returns cryptographically strong random numbers`?

File: operations/secure_random/test_secure_random.py (hunk 1 above)
Comment (justin1121) [reply operations/secure_random/test_secure_random.py]: Tensorflow wouldn't let me do that. I don't think you can have ops with the same name even though they're in a different shared object.

File: tests/test_secure_random.py (hunk 4 above)
Comment (justin1121) [reply tests/test_secure_random.py]: This one tests the wrapper around the secure random functions and the other one is more comprehensive. Could just have one here though.

File: operations/secure_random/test_secure_random.py (hunk 1 above)
Comment (mortendahl) [reply operations/secure_random/test_secure_random.py]: ah I see, makes sense. is it worth using `secure` everywhere then for consistency?

Comment (justin1121) [reply operations/secure_random/test_secure_random.py]: sure, we're exposing the shorter name to the user, `seeded_random_uniform` and `random_uniform`

**Pull request merged**